<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab_Profile</class>
 <widget class="QWidget" name="tab_Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>10</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Account Information:</string>
   </property>
  </widget>
  <widget class="QLabel" name="registeredSinceLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>681</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Registered since:</string>
   </property>
  </widget>
  <widget class="QLabel" name="stocksOwnedLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Stocks Owned:</string>
   </property>
  </widget>
  <widget class="QLabel" name="totalStocksOwnedLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Total Shares Owned:</string>
   </property>
  </widget>
  <widget class="QLabel" name="mostStockShareLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Stock With Most Shares Owned:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
